doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -357,21 +344,7 @@ glspl{component}
 
 \end_inset
 
- interact with each other.
- If a 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
-
-
-\backslash
-gls{component}
-\end_layout
-
-\end_inset
-
- calls another 
+ (or bundles) at a time, some internal 
```

Comment:
Of course the depended components of the specific software have to work. But not the execution of different software (e.g. PDF Reader wants to open URL in a Browser). Maybe this distiction is negligible for us, but I remeber Michael, telling us something like that (Friday meeting)